haskell/ghc_bindist.bzl

```diff
@@ -266,6 +266,14 @@ haskell_toolchain(
         haddock_flags = ctx.attr.haddock_flags,
         repl_ghci_args = ctx.attr.repl_ghci_args,
     )
+
+    if os == "windows":
+        # These libraries cause linking errors on Windows when linking
```

Comment:
Is there an upstream ticket for this?

Reply:
Not that I know. I'll try to come up with a good repro and open a ticket.